src/main/java/com/gooddata/md/visualization/VisualizationObject.java

```diff
@@ -35,15 +36,29 @@
 import java.util.function.Function;
 
 /**
- * Stores complete information about new visualization object that can be stored as MD to md server
+ * Complete information about new visualization object that can be stored as MD to md server.
```

Comment:
Could you reference `Obj` or some of its childs, since it's the class representing the so called `MD object`in SDK?

Reply:
fixed